src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TodoList.css';
+import { Todo } from '../Todo/Todo';
+import { User } from '../User/User';
+
+export const TodoList = ({ todos }) => (
+  <ul className="list list-group">
+    {todos.map(todo => (
+      <li className="list-group-item" key={todo.id}>
+        <Todo title={todo.title} completed={todo.completed} />
+        {' | '}
+        <User user={todo.user.name} />
```

Comment:
Ти ж передаєш тільки ім'я , а не цілого користувача.
```suggestion
        <User userName={todo.user.name} />
```